tanner/emulators/lfi.py

```diff
@@ -1,28 +1,25 @@
 import shlex
 
-from tanner.utils import docker_helper
+from tanner.utils import aiodocker_helper
 from tanner.utils import patterns
 
 
 class LfiEmulator:
     def __init__(self):
-        self.helper = docker_helper.DockerHelper()
+        self.helper = aiodocker_helper.AIODockerHelper()
 
-    async def get_lfi_result(self, container, file_path):
+    async def get_lfi_result(self, file_path):
         # Terminate the string with NULL byte
         if '\x00' in file_path:
             file_path = file_path[:file_path.find('\x00')]
 
-        cmd = 'cat {file}'.format(file=shlex.quote(file_path))
-        execute_result = await self.helper.execute_cmd(container, cmd)
-        # Nulls are not printable, so replace it with another line-ender
-        execute_result = execute_result.replace('\x00', '\n')
-        return execute_result
+        cmd = ["sh", "-c", "cat {file}".format(file=shlex.quote(file_path))]
+        execute_result = await self.helper.execute_cmd(cmd)
```

Comment:
since in lfi we are only read the files, it's better to create only 1 container (pass the same name) and on a shutdown of tanner remove this container

Reply:
As such we don't really need to create a container before to execute some command.

Reply:
Run method creates the containers, yes, but since you don't provide any name, it will create each time a new container, which is not ok.